src/screens/FilterScreen.js

```diff
@@ -122,23 +153,52 @@ class FilterScreen extends Component {
         })
     };
 
+    onFilterScrutinyToggleChange = ({filterToggleKey}) => {
+        this.props.filterToggleLogicUpdated({filterToggleKey});
+    };
+
+    renderToggleSwitch = ({title}) => {
+        const {toggleContainerStyle} = styles;
+        const {filterResultsScrutinyLoose} = this.props;
+
+        if (title === features || title === facilities) {
+            const lowercaseTitle = _.toLower(title);
+
+            return (
+                <View style={toggleContainerStyle}>
+                    <Text>{filter.exactly_these}</Text>
+                    <Switch
+                        onValueChange={() => this.onFilterScrutinyToggleChange({filterToggleKey: lowercaseTitle})}
+                        value={filterResultsScrutinyLoose[lowercaseTitle]}
+                    />
+                    <Text>{filter.any_of_these}</Text>
+                </View>
+            )
+        }
```

Comment:
General purpose render for the toggles. We take their title and use it to both show the display as well as know which one to toggle the logic for in the actions and reducers.